clients/windows/lockbook/Settings.xaml.cs

```diff
@@ -6,26 +6,78 @@
 using Windows.UI.Popups;
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
 using Windows.UI.Xaml.Media.Imaging;
 
 namespace lockbook {
 
     public sealed partial class SignInContentDialog : ContentDialog {
         const ulong BYTE = 1;
-        const ulong KILOBYTES = BYTE* 1000;
-        const ulong MEGABYTES = KILOBYTES* 1000;
-        const ulong GIGABYTES = MEGABYTES* 1000;
-        const ulong TERABYTES = GIGABYTES* 1000;
+        const ulong KILOBYTES = BYTE * 1000;

```

Comment:
https://github.com/lockbook/lockbook/pull/502

should probably use this, hot off the press